File: TP/TP1/Code/4B.cpp
/* Afficher la plus petite de deux valeurs */
#include <iostream>

int main()
{
  int a;
  int b;

  std::cout << "Donne moi un chiffre : " << std::flush;
  std::cin >> a;
  std::cout << "Donne moi un autre chiffre : " << std::flush;
  std::cin >> b;

  if (a < b) {
    std::cout << "La plus petite est : " << a << std::endl;
  } else if (a > b) {
    std::cout << "La plus petite est : " << b << std::endl;
  } else {
    std::cout << "Ils sont identiques !" << std::endl;
  }
  
  return 0;
}

File: TP/TP1/Code/Python/4B.py
#!/usr/bin/python
# Afficher la plus petite de deux valeurs
print('Donne moi une valeur : ', end='')
foo = int(input())
print('Donne moi une autre valeur : ', end='')
bar = int(input())

if foo > bar:
    print(foo, 'est supérieur à', bar)
elif foo < bar:
    print(bar, 'est supérieur à', foo)
else:
    print('Ils sont identiques')
